test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js

```diff
@@ -1,4 +1,4 @@
-/* global describe, it, expect, spyOn */
+/* global describe it expect spyOn beforeAll afterAll */
```

Comment:
Whilst working on eslint/prettier, I found that adding `jasmine` to the env for everything under `tests/` eliminated all these globals.

Reply:
Really? I never got that to work. Might be how I have eslint set up. I'll it.

Reply:
Add an .eslintrc.yml file to the test directory, with the content:

```yaml
env:
  amd: true
  es6: true
  jasmine: true
```

and you should be good to go!

Reply:
Thanks. I added that file.